### PR TITLE
updating from new cosmos spec

### PR DESCRIPTION
**Pull request by seankane-msft**

### CODE REVIEW

File: sdk/tables/azure-data-tables/azure/data/tables/_generated/_azure_table.py [hunk 1]
```diff
@@ -28,7 +28,7 @@ class AzureTable(object):
     :vartype table: azure.data.tables.operations.TableOperations
     :ivar service: ServiceOperations operations
     :vartype service: azure.data.tables.operations.ServiceOperations
-    :param url: The URL of the service account or table that is the target of the desired operation.
+    :param url: The URL of the service account or table that is the targe of the desired operation.
```
Comment (iscai-msft) [inline sdk/tables/azure-data-tables/azure/data/tables/_generated/_azure_table.py:31]: lmao, not super high priority but can you pinig the service team about the swagger / add a fix for this

**Review by iscai-msft — APPROVED** [review]
will approve, just file an issue (doesn't h avet o be high priority) for the service to fix the `target` to `targe` regression

File: sdk/tables/azure-data-tables/azure/data/tables/_generated/_azure_table.py (hunk 1 above)
Comment (seankane-msft) [reply sdk/tables/azure-data-tables/azure/data/tables/_generated/_azure_table.py:31]: Thank you for finding this, making a PR now.

**Pull request merged**